data/examples/declaration/value/function/infix/lenses-out.hs

```diff
@@ -1,9 +1,12 @@
 lenses =
   Just $
     M.fromList $
-      "type" .= ("user.connection" :: Text)
-        # "connection" .= uc
-        # "user" .= case name of
+      "type"
+        .= ("user.connection" :: Text)
+        # "connection"
+        .= uc
+        # "user"
+        .= case name of
```

Comment:
With the default fixity map extracted from Hoogle, `.=` and `#` can't be ordered, so the expected behaviour is to place each operation on a new line as the whole operator chain is multiline